format/spec.md

```diff
@@ -305,6 +305,13 @@ The source column, selected by id, must be a primitive type and cannot be contai
 
 Partition specs capture the transform from table data to partition values. This is used to transform predicates to partition predicates, in addition to transforming data values. Deriving partition predicates from column predicates on the table data is used to separate the logical queries from physical storage: the partitioning can change and the correct partition filters are always derived from column predicates. This simplifies queries because users don’t have to supply both logical predicates and partition predicates. For more information, see Scan Planning below.
 
+Two partition specs are considered equivalent with each other if they have the same number of fields
+and for each corresponding field, the fields have the same source column ID, transform definition
+and partition name. Writers must not create a new parition spec if there already exists a compatible partition
+spec defined in the table.
```

Comment:
Nit: could you remove the newlines to match the formatting of the rest of the doc?

Reply:
I believe this is done now.

Reply:
I believe this should be fixed now.